.github/workflows/continuous-integration.yml

```diff
@@ -379,18 +378,15 @@ jobs:
             .cache/yarn
             node_modules
 
-      - name: Set IS_MASTER_BUILD
+      - name: List changed files
+        id: changed-files
         run: |
           if [[ ${{ github.ref }} == 'refs/heads/master' ]]; then
-              echo "IS_MASTER_BUILD=true" >> $GITHUB_ENV
+            echo ::set-output name=all_changed_files::$(git diff --name-only HEAD~)
```

Comment:
I noticed that with `git diff --name-only HEAD~`, if a file is renamed/moved in a commit, it only shows that the new file path was changed, but not the old one.

e.g. when moving `src/platform/site-wide/banners/index.js` -> `src/platform/new-path/banners/index.js` in a commit:

`git diff --name-only HEAD~` only shows that `src/platform/new-path/banners/index.js` was changed, and not `src/platform/site-wide/banners/index.js`. 

This probably won't be an issue 99% of the time, but if that change was renaming/moving 
`src/platform/site-wide/banners/index.js` -> `src/applications/static-pages/banners/index.js` for example, platform tests wouldn't run, even though a file in `src/platform` was changed. Granted, linting would catch if something was wrong on the import side, it's unknown how that would affect functionality if untested.

I use `git diff-tree --no-commit-id --name-only -r ${{ commit_sha }}` in the [E2E workflow](https://github.com/department-of-veterans-affairs/vets-website/blob/master/.github/workflows/e2e-tests.yml#L57) for getting the changed files on `master`. Using `git diff-tree` seems to include both the source and destination renamed files. Not sure if there's a way to get that functionality with `git diff`.

Reply:
Thanks for reviewing. I don't think it's possible to do this with `git diff` so I updated the PR with `diff-tree`.